migrate/20190415140119_create_comfy_cms_home_carousel_slides.rb

```diff
@@ -0,0 +1,20 @@
+class CreateComfyCmsHomeCarouselSlides < ActiveRecord::Migration
+
+  def change
+    create_table :comfy_cms_home_carousel_slides do |t|
```

Comment:
Here you can probably use `rename_table` instead and after that you can add the boolean field with `add_column`